guides/common-issues.md

```
@@ -190,6 +190,26 @@ Using UTF-8 in PHP arguments doesn't work on Windows for PHP <= 7.0, however it
 
+### Arguments containing newlines are truncated on Windows
+
+On Windows, `wp` is a `.bat` wrapper around `php wp-cli.phar`. Batch files terminate a command at a newline, including a newline inside a quoted argument, because `cmd.exe` splits the line before quoting is evaluated.
+
+Any flags positioned after the multi-line value never reach WP-CLI. The command still succeeds with the arguments it did receive, so there is no error — WordPress silently applies its defaults for the missing values. In the example below the post is created, but as a draft under Uncategorized and authored by the default user, because `--post_category`, `--post_author` and `--post_status` were all dropped:
+
+    wp post create --post_status=publish --post_content="<multi-line content>" --post_category=3 --post_author=2
+
+Pass the content over `STDIN` instead, using `-` as the positional argument. The newline then lives in the piped stream rather than in the command line, so nothing is truncated:
+
+    cat content.html | wp post create - --post_status=publish --post_category=3 --post_author=2
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)